example/prom-collector/go.mod

```diff
@@ -0,0 +1,18 @@
+module go.opentelemetry.io/otel/example/prom-collector
+
+go 1.14
+
+replace (
+	go.opentelemetry.io/otel => ../..
+	go.opentelemetry.io/otel/exporters/metric/prometheus => ../../exporters/metric/prometheus
+	go.opentelemetry.io/otel/exporters/otlp => ../../exporters/otlp
+	go.opentelemetry.io/otel/sdk => ../../sdk
+)
+
+require (
+	go.opentelemetry.io/otel v0.15.0
+	go.opentelemetry.io/otel/exporters/metric/prometheus v0.14.0
+	go.opentelemetry.io/otel/exporters/otlp v0.14.0
```

Comment:
These two requirements should also be at `v0.15.0`.

Reply:
Fixed.